easybuild/easyconfigs/p/PhyML/PhyML-3.3.20170530-goolf-1.7.20.eb

```diff
@@ -0,0 +1,31 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'ConfigureMake'
+
+name = 'PhyML'
+version = '3.3.20170530'
+
+homepage = 'https://github.com/stephaneguindon/phyml'
+description = "Phylogenetic estimation using (Maximum) Likelihood"
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['https://github.com/stephaneguindon/phyml/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['f826726cd56b755be75f923abdf29aca8a9951d6af948cddbab40739a8f99f74']
+
+preconfigopts = ['sh autogen.sh && ', 'sh autogen.sh && ', 'sh autogen.sh && ', 'sh autogen.sh && ']
```

Comment:
@pescobar no need to repeat this, if you just specify it once in a string, the value will be recycled for every configure-build-install cycle

```python
preconfigopts = 'sh autogen.sh && '
```